mmdet3d/models/detectors/base.py

```diff
@@ -60,7 +60,7 @@ def forward(self, return_loss=True, **kwargs):
         else:
             return self.forward_test(**kwargs)
 
-    def show_results(self, data, result, out_dir):
+    def show_results(self, data, result, out_dir, show=False, score_thr=0.3):
```

Comment:
Need to update docstring for `show` and `score_thr`. Besides, please kindly explain why set the default value of `score_thr` to 0.3. Or why not just set `score_dir` to `-1` or `None` to show all results by default for bc-breaking?

Reply:
we need to avoid BC-breaking

Reply:
value `0.3` of `score_thr` is from `mmdetection3d\mmdet3d\apis\test.py\single_gpu_test`'s default parameter `show_score_thr`. I've revised score_thr of detectors/base.py default value to None.